src/lib.rs

```diff
@@ -174,3 +194,59 @@ fn expand_tracing_init() -> Tokens {
   #[cfg(not(feature = "trace"))]
   quote! {}
 }
+
+#[derive(Debug, Default)]
+struct AttributeArgs {
+  default_log_filter: Option<String>,
+}
+
+fn populate_attribute_args(
+  attribute_args: &mut AttributeArgs,
+  attr: &Attribute,
+) -> syn::Result<bool> {
+  if !attr.path().is_ident("test_log") {
+    return Ok(false);
+  }
+
+  let nested_meta = attr.parse_args_with(Meta::parse)?;
+  let Meta::NameValue(name_value) = nested_meta else {
+    return Err(syn::Error::new_spanned(
+      &nested_meta,
+      "Expected NameValue syntax, e.g. 'default_log_filter = \"debug\"'.",
+    ));
+  };
+
+  let Some(ident) = name_value.path.get_ident() else {
+    return Err(syn::Error::new_spanned(
+      &name_value.path,
+      "Expected NameValue syntax, e.g. 'default_log_filter = \"debug\"'.",
+    ));
+  };
+
+  let arg_ref = match ident.to_string().as_ref() {
+    "default_log_filter" => &mut attribute_args.default_log_filter,
+    _ => {
+      return Err(syn::Error::new_spanned(
+        &name_value.path,
+        "Unrecognized attribute, see documentation for details.",
+      ));
+    },
```

Comment:
It seems as if this may be doable without need for explicit `to_string` conversion?
```suggestion
  let arg_ref = if ident == "default_log_filter" {
    &mut attribute_args.default_log_filter
  } else {
    return Err(syn::Error::new_spanned(
      &name_value.path,
      "Unrecognized attribute, see documentation for details.",
    ));
```

Reply:
Ah yeah I guess it was just erroring because I was using a `match`. Multiple `if` + `else if`s would work if we added different attribute types in the future.. updated!